common/lib/request.js

```diff
@@ -1,8 +1,14 @@
 const queryString = require('query-string')
 
+
+const buildAuthHeader = function(token) {
```

Comment:
For consistency maybe use the same shorthand syntax that is used for `getQueryString`?

Reply:
I've moved this into the user service work